src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -920,10 +910,8 @@ export function getNextEvents(currentStatus, details) {
             description: (
               <p>
                 At your hearing, a Veterans Law Judge will ask you questions about your appeal. A
-                transcript of your hearing will be taken and added to your appeal file. The judge
-                will not make a decision about your appeal at the hearing. Learn more about hearings,
-                including how to request a different kind of hearing or withdraw your hearing
-                request.
+                transcript of your hearing will be made and added to your appeal file. The judge
+                will not make a decision about your appeal at the hearing. Learn more about hearings, including how to request a different kind of hearing or withdraw your hearing request.
```

Comment:
@gnakm :thinking: Is there a resource linked to this "Learn more?"

Reply:
Good question @cvalarida, should it link to `/disability-benefits/claims-appeal/hearings/`?